src/client/components/equalizer/equalizer.js

```diff
@@ -1,73 +1,53 @@
 import React, { Component } from 'react';
-import Hark from 'hark';
```

Comment:
If you don’t use it anymore, remove also from package json

Reply:
deleted